db/migrate/20180221161343_create_device_tokens.rb

```diff
@@ -0,0 +1,12 @@
+class CreateDeviceTokens < ActiveRecord::Migration
+  def change
+    create_table :device_tokens do |t|
+      t.integer :user_id, :null => false
+      t.string :token, :null => false
+
+      t.timestamps
+    end
+    add_index :device_tokens, [:user_id, :token]
+    add_index :device_tokens, [:user_id]
```

Comment:
This one is useless due to the left-most thingie. Postgresql can use that one above when only needing user_id